tests/acceptance/pageObjects/phoenixPage.js

```diff
@@ -103,10 +87,18 @@ module.exports = {
     },
     /**
      * Checks to assert the notification bell is not present as well as no notifications are present
+     *
+     * @return boolean
      */
-    assertNoNotifications: function () {
-      return this.waitForElementNotPresent('@notificationBell')
-        .assert.elementNotPresent(this.elements.notificationElement)
+    isNotificationAbsent: async function () {
+      let isAbsent = false
+      await this.waitForElementNotPresent('@notificationBell')
+        .api.elements(
```

Comment:
You are not expecting multiple icons here. So it would be better to use `this.api.element()` instead.

Reply:
done!
:rocket: